src/backend/db/products.js

```diff
@@ -8,23 +8,202 @@ import { v4 as uuid } from "uuid";
+    size: 'medium',
+    categoryName: "toys",
+  },
+  {
+    _id: uuid(),
+    title: "Super Dog Rubber Hole Ball Toy (Large)",
+    tag: "bestseller",
+    price: 199,
+    rating: 4.5,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501622/home/61zQd302BSL._SY450__hqgwsd.jpg',
+    size: 'large',
+    categoryName: "toys",
+  },
+  {
+    _id: uuid(),
+    title: "Red Printed Sweatshirt",
+    tag: "bestseller",
+    price: 199,
+    rating: 4.5,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501410/home/img_4671_awlaj4.jpg',
+    size: 'medium',
+    categoryName: "clothes",
+  },
+  {
+    _id: uuid(),
+    title: "Beboji Argyle Read Sweater For Dogs",
+    tag: "",
+    price: 199,
+    rating: 4.3,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501410/home/img_4568_uoibmt.jpg',
+    size: 'small',
+    categoryName: "clothes",
+  },
+  {
+    _id: uuid(),
+    title: "Navy Sweatshirt with Hoodie",
+    tag: "new",
+    price: 149,
+    rating: 4.9,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501410/home/img_4680_dykvam.jpg',
+    size: 'large',
+    categoryName: "clothes",
+  },
+  {
+    _id: uuid(),
+    title: "FETCHER Blue & Black Premium Dog T-Shirts (Set of 2)",
+    tag: "choice",
+    price: 299,
+    rating: 5,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501679/home/61bSqyBA5_L._SY450__duw9vx.jpg',
+    size: 'small',
+    categoryName: "clothes",
+  },
+  {
+    _id: uuid(),
+    title: "Lana Paws Adjustable Tuxedo Dog Bandana/Scarf",
+    tag: "bestseller",
+    price: 399,
+    rating: 4,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501811/home/A1iXBU7wrWL._AC_UL320__mlzn5i.jpg',
+    size: 'large',
+    categoryName: "clothes",
+  },
+  {
+    _id: uuid(),
+    title: "Dear Pet Blooming Yellow Dog Harness",
+    tag: "bestseller",
+    price: 299,
+    rating: 4.7,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501410/home/cards1_zi4fie.png',
+    size: 'medium',
+    categoryName: "leash",
+  },
+  {
+    _id: uuid(),
+    title: "Dear Pet Blooming Yellow Dog Leash",
+    tag: "new",
+    price: 199,
+    rating: 3,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501410/home/floral-combo_wclxym.webp',
+    size: 'large',
+    categoryName: "leash",
+  },
+  {
+    _id: uuid(),
+    title: "Pet Me Luxury Stylish Rope Leash",
+    tag: "",
+    price: 299,
+    rating: 4,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501812/home/51lkFg0sCtL._AC_UL320__y4q9iq.jpg',
+    size: 'small',
+    categoryName: "leash",
+  },
+  {
+    _id: uuid(),
+    title: "Retractable Dog Leash, 26 Foot",
+    tag: "choice",
+    price: 499,
+    rating: 4,
+    img: 'https://res.cloudinary.com/do5ez3xws/image/upload/v1647501812/home/614XeyHUIRL._SY450__xnjby4.jpg',
```

Comment:
Had a query on my side are you hosting these images on Cloudinary?Or are you using pre made links?Want to know about the thought process how you designed the data